docs/api/layer-api.md

```diff
@@ -125,6 +125,94 @@ let layer = document.querySelector('map-layer');
 let opacity = layer.opacity;
 ```
 
+### extent
+
+Read-only.  Returns the upper left and lower right coordinates of the 
```

Comment:
I think it would be good to keep the properties documentation consistent with how other properties are documented, for ex, for checked:
`HTMLLayerElement.checked is a boolean read/write value that`